decoding_morse.rb

```diff
@@ -0,0 +1,33 @@
+# Create a method to decode a Morse code character
+def decode_char(morse_code)
+  morse_code_dict = {
+    '.-' => 'A',
+    '-...' => 'B',
+    '-.-.' => 'C', '-..' => 'D',
+    '.' => 'E', '..-.' => 'F', '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+
+  morse_code_dict[morse_code.strip] || 'UNKNOWN'
+end
+
+# Create a method to decode an entire word in Morse code
+def decode_word(morse_code)
+  morse_code.split.map { |morse_code_char| decode_char(morse_code_char) }.join
+end
+
+# Create a method to decode the entire message in Morse code
+def decode_message(morse_message)
+  words = morse_message.split('   ')
+  decoded_words = words.map do |_word|
+    characters = word.chars
+    decoded_characters = characters.map { |char| decode_char(char) }
+    decoded_characters.join
+  end
+
+  decoded_words.join('   ')
+end
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :smile: :wave: 

- [ ] According to the project requirement, you are meant to run this test to see your decode morse code in action.

```sh
p decode_char('.-')
p decode_word('-- -.--')
p decode_message("-- -.--   -. .- -- .")
p decode_message(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```
-  [ ] When I ran these, I got an error in the console, I'll suggest you go through your methods and make sure everything is implemented correctly, and then run the code to make sure the result you get is what's expected. :smile: :small_airplane: :wave: 

- - Screenshot for reference:
![Screenshot from 2023-08-30 11-59-13](https://github.com/RileyManda/morse-code/assets/107632780/73370a66-2903-435d-a29a-dbb7c2bc55cf)